### PR TITLE
DISTPG-316 Release notes 14.0

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: docs/release-notes-v14.0.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+|[`pgAudit set user`](https://github.com/pgaudit/set_user)| 2.0.1|  provides an additional layer of logging and control when unprivileged users must escalate themselves to superuser or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.34    | a backup and restore solution for PostgreSQL       |
+|[`pgBadger`](https://github.com/darold/pgbadger) | 11.5       | a fast PostgreSQL Log Analyzer.|
+|[`pgBouncer`](https://www.pgbouncer.org/) | 1.15.0 | lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.6   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)| 0.9.2 - Beta1 | collects and aggregates statistics for PostgreSQL and provides histogram information.       |
+|[`wal2json`](https://github.com/eulerto/wal2json) |2.3        | a PostgreSQL logical decoding JSON output plugin.|
+
+Percona Distribution for PostgreSQL also includes the ETCD packages which are used for Patroni cluster setup. These packages are available for the following operating systems:
```
Comment (LenzGr) [inline docs/release-notes-v14.0.md]:
```suggestion
Percona Distribution for PostgreSQL also includes supplemental ETCD packages which can be used for setting up Patroni clusters. These packages are available for the following operating systems:
```

File: docs/release-notes-v14.0.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+| Debian 9 ('stretch')| `etcd`               | A consistent, distributed key-value store|
+|                     | `python3-etcd`       | A Python client for ETCD     |
+
+                                                      
+Percona Distribution for PostgreSQL is also shipped with the [libpq](https://www.postgresql.org/docs/14/libpq.html) library. It contains "a set of
+library functions that allow client programs to pass queries to the PostgreSQL
+backend server and to receive the results of these queries." [^1]
+
+
+[^1]: https://www.postgresql.org/docs/13/libpq.html
```
Comment (LenzGr) [inline docs/release-notes-v14.0.md]: I think that footnote can be removed? It still points to version 13 of the docs, and we have a link in the paragraph above.

Comment (nastena1606) [reply docs/release-notes-v14.0.md]: Removed

File: docs/release-notes-v14.0.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* Stored procedures can accept OUT parameters.
+* The [libpq](https://www.postgresql.org/docs/14/libpq.html) library now supports the pipeline mode. Previously, the client applications waited for a transaction to be completed before sending the next one. The pipeline mode allows the applications to send multiple transactions at the same time thus boosting performance.
+* Large transactions are now streamed to subscribers in-progress, thus increasing the performance. This improvement applies to logical replication API as well.
+* LZ4 compression is added for [TOAST](https://www.postgresql.org/docs/current/storage-toast.html) operations. This speeds up large data processing.
+* SCRAM is made the default authentication mechanism. This mechanism improves security and simplifies regulatory compliance for data security.
+
+!!! seealso
+
+    * [PostgreSQL 14 release notes](https://www.postgresql.org/docs/release/14/)
+    * Percona Blog: [PostgreSQL 14 – Performance, Security, Usability, and Observability](https://www.percona.com/blog/postgresql-14-performance-security-usability-and-observability/)
```
Comment (LenzGr) [inline docs/release-notes-v14.0.md:51]: We have some additional PG 14 blog posts that we could include here:

- [Using the Range and the New Multirange Data Type in PostgreSQL 14](https://www.percona.com/blog/using-the-multirange-data-type-in-postgresql-14/)
- [Using New JSON Syntax in PostgreSQL 14 – Update](https://www.percona.com/blog/using-new-json-syntax-in-postgresql-14-update/)

File: docs/release-notes-v14.0.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+* `llvm` 11.0.0 packages for Red Hat Enterprise Linux 8 / CentOS 8. This fixes compatibility issues with LLVM from upstream.
+* supplemental `ETCD` packages which can be used for setting up Patroni clusters. These packages are available for the following operating systems:
+
+|  Operating System   | Package              | Version | Description        |
+| ------------------- | ---------------------| --------| ------------------ |
+| CentOS 7            |`python3-python-etcd` | 0.4.3   | A Python client for ETCD     |
+| CentOS 8            | `etcd`               | 3.3.11  | A consistent, distributed key-value store|
+|                     | `python3-python-etcd`| 0.4.3   | A Python client for ETCD     |
+| Debian 9 ('stretch')| `etcd`               | 3.3.11  | A consistent, distributed key-value store|
+|                     | `python3-etcd`       | 0.4.5   | A Python client for ETCD     |
```
Comment (LenzGr) [inline docs/release-notes-v14.0.md]: @EvgeniyPatlan can you please review and double check these version numbers? Thank you!

**Review by LenzGr — APPROVED** [review]
Looks good to me, but I'd like to get @EngineeredVirus approval on this one as well.

File: docs/release-notes-v14.0.md (hunk 3 above)
Comment (nastena1606) [reply docs/release-notes-v14.0.md:51]: Added

File: docs/release-notes-v14.0.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    * Percona Blog: [PostgreSQL 14 – Performance, Security, Usability, and Observability](https://www.percona.com/blog/postgresql-14-performance-security-usability-and-observability/)
+    * [Using the Range and the New Multirange Data Type in PostgreSQL 14](https://www.percona.com/blog/using-the-multirange-data-type-in-postgresql-14/)
+    * [Using New JSON Syntax in PostgreSQL 14 – Update](https://www.percona.com/blog/using-new-json-syntax-in-postgresql-14-update/)
+
+The following is the list of extensions available in Percona Distribution for PostgreSQL.
+
+| Extension           | Version        | Description                  |
+| ------------------- | -------------- | ---------------------------- |
+| [Patroni](https://patroni.readthedocs.io/en/latest/) | 2.1.0 | a HA (High Availability) solution for PostgreSQL |
+| [Pgaudit](https://www.pgaudit.org/)             | 1.6.0 Beta | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
```
Comment (EvgeniyPatlan) [inline docs/release-notes-v14.0.md]: why Beta?

File: docs/release-notes-v14.0.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+|[`pgAudit set user`](https://github.com/pgaudit/set_user)| 3.0.0|  provides an additional layer of logging and control when unprivileged users must escalate themselves to superuser or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.36    | a backup and restore solution for PostgreSQL       |
+|[`pgBadger`](https://github.com/darold/pgbadger) | 11.5       | a fast PostgreSQL Log Analyzer.|
+|[`pgBouncer`](https://www.pgbouncer.org/) | 1.16.0 | lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.7   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)| 0.9.2 - Beta1 | collects and aggregates statistics for PostgreSQL and provides histogram information.       |
```
Comment (EvgeniyPatlan) [inline docs/release-notes-v14.0.md]: 1.0.0-Beta2

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: docs/release-notes-v14.0.md (hunk 6 above)
Comment (nastena1606) [reply docs/release-notes-v14.0.md]: Fixed

**Review by EvgeniyPatlan — APPROVED** [review]

**Comment by nastena1606** [comment]
This was an internal release. Publicly released version 14.1

**Pull request closed without merging**